source/lib/stoprint/api.js

```diff
@@ -40,28 +42,71 @@ export async function logIn(
 		}))
 
 	if (result.error) {
-		return 'The print server seems to be having some issues'
+		return 'The print server seems to be having some issues.'
 	}
 
 	if (!result.value.success) {
-		return 'The username and password appear to be invalid'
+		return 'The username and password appear to be invalid.'
```

Comment:
Maybe `Your username and password appear to be invalid.`?

Reply:
I would prefer an “or” over an “and”

Reply:
I concur with Drew.